src/OmniSharp.LanguageServerProtocol/Handlers/OmniSharpCompletionHandler.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace OmniSharp.LanguageServerProtocol.Handlers
 {
-    class OmniSharpCompletionHandler : CompletionHandler
+    class OmniSharpCompletionHandler : CompletionHandlerBase
```

Comment:
Man, this whole handler needs to be ported to the new one, doesn't it.

Reply:
yes

Reply:
thankfully the changed merged in cleanly.